### DISCUSSION TITLE
PeaNUT LXC spinning circle

### DISCUSSION THREAD

**Discussion by asylumexp** [tteck/Proxmox#3220] [Help Needed] [answered]
Trying to get PeaNUT working but encountering spinning circle on load.
![image](https://github.com/tteck/Proxmox/assets/54530346/df6c7f9b-73fb-4d8e-b316-b31c67ba9478)

This is presumably because NUT isn't configured or something isn't running as the service is dead on LXC creation.
```
○ nut-monitor.service - Network UPS Tools - power device monitor and shutdown controller
     Loaded: loaded (/lib/systemd/system/nut-monitor.service; enabled; preset: enabled)
     Active: inactive (dead) since Mon 2024-06-17 18:46:00 AEST; 9min ago
   Duration: 2ms
   Main PID: 5182 (code=exited, status=0/SUCCESS)
        CPU: 701us

Jun 17 18:46:00 peanut systemd[1]: Started nut-monitor.service - Network UPS Tools - power device monitor and shutdown controller.
Jun 17 18:46:00 peanut nut-monitor[5182]: upsmon disabled, please adjust the configuration to your needs
Jun 17 18:46:00 peanut nut-monitor[5182]: Then set MODE to a suitable value in /etc/nut/nut.conf to enable it
Jun 17 18:46:00 peanut systemd[1]: nut-monitor.service: Deactivated successfully.
```

also have GitHub issue from person who made the script, might have something to do with it Brandawg93/PeaNUT#35

**Comment by asylumexp** [comment, marked as answer]
My mistake, missed the disclaimer on the script page on the website

**Reply by gezuka77** [reply]
What "disclaimer" you mean?
For now I have deployed this in a docker-compose.yml and changing NUT_HOST=NUTSERVERIP solved the issue. I have a separate TrueNAS server which acts as a NUTSERVER where the UPS is physically connected.

**Reply by Mellowlynx** [reply]
Probably this one
![image](https://github.com/tteck/Proxmox/assets/2348339/b23796a1-a929-4475-be52-7016e33c1dc2)

**Reply by gezuka77** [reply]
I see, thanks!

**Reply by Rocketplanner83** [reply]
What files need to be configured?
